test/jasmine/tests/sankey_d3_sankey_test.js

```diff
@@ -0,0 +1,130 @@
+// var Plotly = require('@lib/index');
+// var attributes = require('@src/traces/sankey/attributes');
+
+var d3sankey = require('d3-sankey');
+
+var data = {
+    'nodes': [{
+        'node': 0,
+        'name': 'node0'
+    }, {
+        'node': 1,
+        'name': 'node1'
+    }, {
+        'node': 2,
+        'name': 'node2'
+    }, {
+        'node': 3,
+        'name': 'node3'
+    }, {
+        'node': 4,
+        'name': 'node4'
+    }],
+    'links': [{
+        'source': 0,
+        'target': 2,
+        'value': 2
+    }, {
+        'source': 1,
+        'target': 2,
+        'value': 2
+    }, {
+        'source': 1,
+        'target': 3,
+        'value': 2
+    }, {
+        'source': 0,
+        'target': 4,
+        'value': 2
+    }, {
+        'source': 2,
+        'target': 3,
+        'value': 2
+    }, {
+        'source': 2,
+        'target': 4,
+        'value': 2
+    }, {
+        'source': 3,
+        'target': 4,
+        'value': 4
+    }]
+};
+
+
+describe('d3-sankey', function() {
```

Comment:
Similarly here, can you move this describe to sankey_test.js and :hocho: this file?

Reply:
Done in 4539d285285ebb22d6d2ce9ddd300e9e1de03b60